### PR TITLE
Improving LnF of section nav bar

### PR DESCRIPTION
**Pull request by davidwatkins73**
#2656

### CODE REVIEW

**Comment by davidwatkins73** [comment]
@kamransaleem - pls review

File: waltz-ng/client/dynamic-section/components/dynamic-section-navigation/dynamic-section-navigation.js [hunk 1]
```diff
@@ -19,18 +19,24 @@ import _ from "lodash";
...
     onSelect: '<',
     offset: '@?'
 };
 
 
 const initialState = {
+    sections: [],
     offset: 250,
     stickyVisible: false,
+    sectionClasses: {},
```
Comment (kamransaleem) [inline waltz-ng/client/dynamic-section/components/dynamic-section-navigation/dynamic-section-navigation.js]: remove

File: waltz-ng/client/dynamic-section/components/dynamic-section-navigation/dynamic-section-navigation.js [hunk 2]
```diff
@@ -57,6 +66,24 @@ function controller($scope,
             .off("scroll", scrollListener);
     };
 
+    vm.$onChanges = () => {
+        colorScale
+            .domain([0, vm.availableSections.length / 2.5]);
```
Comment (kamransaleem) [inline waltz-ng/client/dynamic-section/components/dynamic-section-navigation/dynamic-section-navigation.js]: const perhaps

File: waltz-ng/client/dynamic-section/components/dynamic-section-navigation/dynamic-section-navigation.scss [hunk 3]
```diff
@@ -21,39 +21,55 @@
...
\ No newline at end of file
+        }
+
+        li:not(:first-child) {
+            border-left: 1px solid #ccc;
+        }
+
+        .wdsn-option {
+            &:hover {
+                //background-color: #f4f4f4;
```
Comment (kamransaleem) [inline waltz-ng/client/dynamic-section/components/dynamic-section-navigation/dynamic-section-navigation.scss]: kill

**Pull request merged**